index.js

```diff
@@ -178,7 +178,7 @@
 //. //
 //. //   The value at position 1 is not a member of ‘FiniteNumber’.
 //. //
-//. //   See https://github.com/sanctuary-js/sanctuary-def/tree/v0.11.0#FiniteNumber for information about the sanctuary-def/FiniteNumber type.
+//. //   See https://github.com/sanctuary-js/sanctuary-def/tree/v0.11.1#FiniteNumber for information about the sanctuary-def/FiniteNumber type.
```

Comment:
By "the rest", do you mean the rest of the lines in this error message or the rest of the lines in the file?